api/user/routes.ts

```diff
@@ -1,11 +1,12 @@
 import * as express from 'express';
-import { confirmUserExclusion, getUserExclusions } from '../exclusions/exclusionService';
+import { confirmUserExclusion, deleteUserExclusion, getUserExclusions } from '../exclusions/exclusionService';
 import { getUserDetails } from './index';
 
 const router = express.Router({ mergeParams: true });
 
 router.use('/details', getUserDetails);
 router.get('/exclusions', getUserExclusions);
 router.post('/exclusions/confirm', confirmUserExclusion);
+router.post('/exclusions/delete', deleteUserExclusion);
```

Comment:
query, what is the difference between the two?
api/exclusions/routes.ts
and
api/user/routes.ts

Reply:
I will look into this

Reply:
@rayliangatos and @connorpgpmcelroy - I also saw this "duplicate" `routes.ts` when I was working on the Delete Exclusion page... the `api/user/routes.ts` doesn't appear to be used?

Reply:
actually it was the opposite @Daniel-Lam, the api/exclusions/routes.ts was not being used - api/user/routes.ts was being used. I've pushed that change in

Reply:
@connorpgpmcelroy - ah, OK... fair enough. I'll mark this conversation as resolved.